pyaedt/modeler/Object3d.py

```diff
@@ -3197,3 +3197,232 @@ def set_net_name(self, netname=""):
         """
         vMaterial = ["NAME:Net", "Value:=", netname]
         return self.change_property(vMaterial)
+
+
+class Point(object):
+    """Manages point attributes for the AEDT 3D Modeler.
+
+    Parameters
+    ----------
+    primitives : :class:`pyaedt.modeler.Primitives3D.Primitives3D`
+        Inherited parent object.
+    name : str
+        Name of the point.
+
+    Examples
+    --------
+    Basic usage demonstrated with an HFSS design:
+
+    >>> from pyaedt import Hfss
+    >>> aedtapp = Hfss()
+    >>> primitives = aedtapp.modeler.primitives
+
+    Create a point, to return an :class:`pyaedt.modeler.Object3d.Point`.
+
+    >>> point = primitives.create_point([30, 30, 0], "my_point", (0, 195, 255))
+    >>> my_point = primitives.points_by_name[point.name]
+    """
+
+    def __init__(self, primitives, name):
+        self._name = name
+        self._point_coordinate_system = "Global"
+        self._color = None
+        self._position = None
+        self._primitives = primitives
+        self._all_props = None
+
+    @property
+    def m_Editor(self):
+        """Pointer to the oEditor object in the AEDT API. This property is
+        intended primarily for use by FacePrimitive, EdgePrimitive, and
+        VertexPrimitive child objects.
+
+        Returns
+        -------
+        oEditor COM Object
+
+        """
+        return self._primitives._oeditor
+
+    @property
+    def logger(self):
+        """Logger."""
+        return self._primitives.logger
+
+    @property
+    def name(self):
+        """Name of the object as a string value.
+
+        Returns
+        -------
+        str
+           Name of object as a string value.
+
+        References
+        ----------
+
+        >>> oEditor.GetPropertyValue
+        >>> oEditor.ChangeProperty
+
+        """
+        return self._name
+
+    @name.setter
+    def name(self, point_name):
+        if point_name not in self._primitives.points_by_name.keys:
+            if point_name != self._name:
+                name_property = []
+                name_property.append("NAME:Name")
+                name_property.append("Value:=")
+                name_property.append(point_name)
+                changed_property = ["NAME:ChangedProps", name_property]
+                property_servers = ["NAME:PropServers"]
+                property_servers.append(self._name)
+                point_tab = ["NAME:Geometry3DPointTab", property_servers, changed_property]
+                all_tabs = ["NAME:AllTabs", point_tab]
+                _retry_ntimes(10, self._primitives._oeditor.ChangeProperty, all_tabs)
+                self._name = point_name
+                self._primitives.cleanup_objects()
+        else:
+            # TODO check for name conflict
```

Comment:
Here I would suggest, at the moment, a logger.warning that name exists

Reply:
Done.